flooder/src/main.rs

```diff
@@ -146,18 +148,37 @@ fn flood(
     num_threads: usize,
     status: XtStatus,
     histogram: &Arc<Mutex<HdrHistogram<u64>>>,
+    rate_limit: Option<(u64, u64)>,
 ) {
-    let transactions_per_batch = txs.len() / num_threads;
-    txs.par_chunks(transactions_per_batch).for_each(|batch| {
-        println!("Sending a batch of {} transactions", &batch.len());
-        batch.iter().enumerate().for_each(|(index, tx)| {
-            send_tx(
-                pool.get(index % pool.len()).unwrap(),
-                tx,
-                status,
-                Arc::clone(histogram),
-            )
-        })
+    let (transactions_in_interval, interval_dr) = rate_limit
+        .map_or((txs.len(), Duration::from_secs(0)),
```

Comment:
Are these defaults going to flood the log with messages from line 180? If so, refactor it slightly.

Reply:
Only one message will appear